enh(broker-developer): new lua command added: get_check_command() (#2370)
centreon/centreon-documentation: versioned_docs/version-23.04/developer/developer-broker-stream-connector.md

10.  ``get_metric_mapping(metric_id)`` that gets from the cache the
    metric mapping object of the given metric id. The result is a table
    containing two keys, ``metric_id`` and ``index_id``.
11. ``get_service_description(host_id,service_id)`` that gets from the cache the
    service description of the given pair host_id / service_id. This function
    returns a string or *nil* otherwise.
12. ``get_servicegroup_name(id)`` that gets from the cache the service group name*
    of the given id. This function returns a string or *nil* otherwise.
13. ``get_servicegroups(host_id, service_id)`` that gets the list of service
    groups containing the service corresponding to the pair *host_id* /
    *service_id*. The return value is an array of objects, each one containing
    two fields, *group_id* and *group_name*.
14. ``get_notes(host_id[,service_id])`` that gets the notes configured in the
    host or service. The *service_id* is optional, if given we want notes from
    a service, otherwise we want notes from a host. If the object is not found
    in cache, *nil* is returned.
15. ``get_notes_url(host_id[, service_id])`` that gets the notes url configured

>     in the host or service. The *service_id* is optional, if given we want

Reviewer: ```suggestion
    in the host or service. The *service_id* is optional. If given, you obtain
```

>     *notes url* from a service, otherwise we want it from a host. If the object

Reviewer: ```suggestion
    *notes url* for a service, otherwise for a host. If the object
```

>     is not found in cache, *nil* is returned.

Reviewer: ```suggestion
    is not found in the cache, *nil* is returned.
```

> 16. ``get_action_url(host_id)`` that gets the action url configured in the host

Reviewer: ```suggestion
16. ``get_action_url(host_id)`` that gets the action URL configured in the host
```

>     or service. The *service_id* is optional, if given we want *action url* from

Reviewer: ```suggestion
    or service. The *service_id* is optional. If given, you obtain the *action url* for
```

>     a service, otherwise we want it from a host. If the object is not found in

Reviewer: ```suggestion
    a service, otherwise for a host. If the object is not found in the
```

    cache, *nil* is returned.
17. ``get_severity(host_id[,service_id])`` that gets the severity of a host or

>     a service. If you only provide the *host_id*, we suppose you want to get

Reviewer: ```suggestion
    a service. If you only provide the *host_id*, you get
```

>     a host severity. If a host or a service does not have any severity, the

Reviewer: ```suggestion
    a host severity. If a host or a service do not have any severity, the
```

3.  ``get_bvs(ba_id)`` that gets all the *bv* containing the *ba* of id *ba_id*.
    This function returns an array of *bv* *ids*, potentially empty if no *bv*
    are found.

> 4.  ``get_check_command(host_id[, service_id])`` that gets the check command

Reviewer 2: Why are there double back-quotes in this file instead of single back-quotes?
